KarhooUISDK/UIComponents/TermsConditionsView/TermsConditionsView.swift

```diff
@@ -70,13 +71,21 @@ public final class TermsConditionsView: UIView, UITextViewDelegate {
     }
     
     public func setBookingTerms(supplier: String?, termsStringURL: String) {
-        termsTextView.attributedText = TermsConditionsStringBuilder()
-            .bookingTermsCopy(supplierName: supplier,
-                              termsURL: convert(termsStringURL))
+        let text = TermsConditionsStringBuilder()
+            .bookingTermsCopy(
+                supplierName: supplier,
+                termsURL: convert(termsStringURL)
+            )
+        setText(text)
     }
     
     public func setToRegistrationTerms() {
-        termsTextView.attributedText = TermsConditionsStringBuilder().registrationTermsCopy()
+        setText(TermsConditionsStringBuilder().registrationTermsCopy())
+    }
+
+    private func setText(_ attributedText: NSAttributedString) {
+        termsTextView.attributedText = attributedText
+        termsTextView.accessibilityLabel = attributedText.string.replacingOccurrences(of: "|", with: ".")
```

Comment:
I'd replace the `.` with `and` as in "you agree to the Terms and Consditions **and** Privacy Policy". This would require a new key in Phrase for "and" so that it would be translated. Maybe check with Olivier to see what he prefers.

Reply:
And require localisation. We can leave it as `.` I think

Reply:
On the other hand using `and` can has unexpected outcome in some languages. `.` is read just like longer pause. This is the least risky approach, I think.